RLTest/__main__.py

```diff
@@ -417,7 +423,14 @@ def __init__(self):
         self.loader = TestLoader()
         if self.args.test:
             self.loader.load_spec(self.args.test)
-        else:
+        if self.args.tests_file:
+            for f in self.args.tests_file:
+                with open(f, 'r') as file:
```

Comment:
```
if self.args.test_file is not None and os.path.exists(self.args.test_file):
    for fname in self.args.tests_file:
        with open(fname) as fp:
            [self.loader.load_spec(line.strip()) for line in fp.readlines() if line.strip() not in ['#', '']]
```

Reply:
`test-file` is actually tests_files - this is a list of files.
IMO the comprehension here makes it hard to understand the code.
Added exception handling.